bin/wasm-build

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+#
+# wasm-build — build all known WASM-supporting crates
```

Comment:
```suggestion
# wasm-build — build a crate as WASM32
```

This requires manually specifying the crate, right?

Reply:
Yes. Holdover from a previous iteration of this script.